examples/rest_api_usecases/update_sobject/README.md

```diff
@@ -0,0 +1,28 @@
+# Use REST API 
+
+This example demonstrates how to invoke REST API of Salesforce to update sObjects.
+
+## Prerequisites
+
+### 1. Set up
+Refer to the setup guide in [ReadMe](../../../README.md) for necessary credentials.
+
+### 2. Configuration
+
+Configure Salesforce API credentials in Config.toml in the example directory:
+
+```toml
+clientId = "<BASE_URL>"
+clientSecret = "<CLIENT_SECRET>"
+refreshToken = "<REFRESH_TOKEN>"
+refreshUrl = "<REFRESH_URL>"
+baseUrl = "<BASE_URL>"
+```
+
+## Run the Example
```

Comment:
```suggestion
## Run the example
```

Reply:
Updated in [Update code according to comments](https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/322/commits/637bff31ea6fdcb76c54d48b8f7054b644e6d85a)